mkdocs.yml

```diff
@@ -138,7 +138,8 @@ nav:
           - Credentials: pages/configure/credentials.md
       - CATALOG DATA:
           - Introduction: pages/catalog-data/introduction.md
-          - Kedro data catalog: pages/catalog-data/data_catalog.md
+          - Data catalog: pages/catalog-data/data_catalog.md
+          - Kedro Data catalog: pages/catalog-data/kedro_data_catalog.md
```

Comment:
@ElenaKhaustova @ravi-kumar-pilla I couldn't figure out with the merging if we should keep just one or both, so kept both for now.